numpy/core/__init__.py

```diff
@@ -75,8 +75,10 @@
 from . import fromnumeric
 from .fromnumeric import *
 from . import defchararray as char
+from . import records
```

Comment:
It's already imported in the line below, exposing it under two names seems wrong. Better to adjust the calling code like `np.core.records` -> `np.core.rec` (or a direct import and not use the fully-qualified name at all, like for `_byteorderconv` in `ma/mrecords.py`

Reply:
I worried about the two names also, `rec` and `records`.  But the standard import exposes two names, and while the numpy `__init__` code mostly uses records, the `rec` is included in an `__all__` so that would be a public API change.  All we're doing is making the lazyloader version have the same names as the standard version.  If we don't provide both names then we're breaking current behavior.

Reply:
I guess the difference is that we have `np.rec` but not `np.records`. `np.rec` is the public namespace; everything else is private anyway. I checked that this change does not cause `np.records` to appear, so we should be okay with this.